### PR TITLE
Bugfix: `about me` not saved on signup

### PR DESCRIPTION
**Pull request by roschaefer**
## 🍰 Pullrequest

Our dear @Hardy-Peaceglobal has reported this issue. This should update the user's `about` attribute on signup.

### Issues

- there is no issue for that yet

### Todo

- [X] None

### CODE REVIEW

File: backend/src/seed/factories/users.js [hunk 1]
```diff
@@ -24,7 +24,7 @@ export default function create() {
       }
       args = await encryptPassword(args)
       const user = await neodeInstance.create('User', args)
-      const email = await neodeInstance.create('EmailAddress', { email: args.email })
+      const email = await factoryInstance.create('EmailAddress', { email: args.email })
```
Comment (mattwr18) [inline backend/src/seed/factories/users.js]: what's the difference between `factoryInstance` and `neodeInstance`?

Comment (roschaefer) [reply backend/src/seed/factories/users.js]: We might want to refactor this with https://github.com/aexmachina/factory-girl (although the repo looks rather old).

`factoryInstance` is a reference to the other factories. If you would do `factoryInstance.create(..)` that would give you all the defaults and also setup associations. If you use `neodeInstance` you would end up with a `User` without `EmailAddress` etc.

Comment (mattwr18) [reply backend/src/seed/factories/users.js]: ahhh, ok! great... thanks

**Comment by mattwr18** [comment]
great job @roschaefer 
and just at the threshold of merging without multiple `PR` reviews!!
:stuck_out_tongue_closed_eyes:

**Pull request merged**